Mathlib/Algebra/Order/Monoid/Canonical/Defs.lean

```diff
@@ -187,32 +187,16 @@ end Preorder
 section PartialOrder
 variable [PartialOrder α] [CanonicallyOrderedMul α] {a b c : α}
 
-@[to_additive]
-theorem bot_eq_one : (⊥ : α) = 1 :=
-  le_antisymm bot_le (one_le ⊥)
-
-@[to_additive (attr := simp)]
-theorem le_one_iff_eq_one : a ≤ 1 ↔ a = 1 :=
-  (one_le a).le_iff_eq
-
-@[to_additive]
-theorem one_lt_iff_ne_one : 1 < a ↔ a ≠ 1 :=
-  (one_le a).lt_iff_ne.trans ne_comm
-
-@[to_additive]
-theorem one_lt_of_ne_one (h : a ≠ 1) : 1 < a :=
-  one_lt_iff_ne_one.2 h
+@[to_additive] theorem bot_eq_one : (⊥ : α) = 1 := rfl
```

Comment:
Can this be moved to an earlier section? I think `section LE` above would work.

Reply:
Nice catch, I've moved it just below the `OrderBot` instance.